### PR TITLE
Removed unneeded boilerplate

### PR DESCRIPTION
**Pull request by GrantMLambert**
This will remove unneeded boilerplate according to the specifications in issue #91.

### CODE REVIEW

**Review by lugenx — COMMENTED** [review]
looks good to me, thank you, i will wait for @rcamach7 approval before merging it.

**Comment by rcamach7** [comment]
Looks great! Thank you @GrantMLambert

**Review by rcamach7 — APPROVED** [review]

**Review by lugenx — APPROVED** [review]

**Pull request merged**
